### DISCUSSION TITLE
BCG variants

### DISCUSSION THREAD

**Discussion by harrismia** [jodyphelan/TBProfiler#221] [Q&A]
Hello, 

   We are using TB Profiler version 4.2.0 to profile some BCG samples.  Can we rely on the variant calls and main/sub lineage reported by TB Profiler for BCG samples?    I believe TB Profiler is using the standard H37Rv reference.  Is this sufficient for profiling BCG samples? 

 Thanks!

**Comment by jodyphelan** [comment]
Hi @harrismia 

The genomes are pretty similar, for example, _rpoB_ differs by one SNP and _katG_ differs by 3. So reads should be aligned fine to most of the drug resistance genes (though I haven't checked them all!). 

That means that the variant calls should all be fine and the lineage assignemnts should all be fine too. Just to note that the default library shipped with v4.2.0 comes with a slightly outdated barcode which classifies all bovis isolates as the same sublineage. If you want BCG (and other livestock associated sublineages) to be classified seperately, then use the updated tbdb library by running `tb-profiler update_tbdb`. The new lineages come from https://open-research-europe.ec.europa.eu/articles/1-100

**Reply by harrismia** [reply]
Hi @jodyphelan , 

       Thanks very much for this information!   We did execute tb-profiler update_tbdb and seem to get the same version of tbdb: 

Database version: tbdb_47daa08_jodyphelan <jody.phelan@lshtm.ac.uk>_Fri Apr 29 10:18:53 2022 +0100

   This version does classify different BCG sublineages so I am guessing that this is the correct version of the database.  Please let me know if otherwise.  

     Thanks a lot for all of your help!

**Reply by jodyphelan** [reply]
Yes that is the correct version!
